api/versioninfo.php

```diff
@@ -0,0 +1,117 @@
+<?php
+  /**
+    Finds the latest versions of downloads and returns a JSON blob with the data. The first *.download_info file in each folder will be checked.
```

Comment:
Have added the other folders; try again. It's going to give you a blank result, though :)